src/main/java/com/esri/core/geometry/ogc/OGCPoint.java

```diff
@@ -77,6 +79,13 @@ public String geometryType() {
 		return "Point";
 	}
 
+	@Override
+	public long estimateMemorySize()
+	{
+		return SIZE_OF_OGC_POINT + (point != null ? point.estimateMemorySize() : 0) +
+				(esriSR != null ? esriSR.estimateMemorySize() : 0);
```

Comment:
In a good implementation, the spatial reference instance should be shared between all instances of geometry coming the same dataset, so usually the esriSR size would not contribute to OGC types, but we don't enforce that on geometry side right now.

Reply:
@stolstov Sergey, would you recommend keeping this or removing and hoping esriSR is actually shared?

Reply:
@mbasmanova Let's exclude it, and add a doc to OGCGeometry.estimateMemorySize, mentioning that the size of spatial reference is not included, because we assume that it is shared.